### PR TITLE
PowerToys PowerRename random values feature docs

### PR DESCRIPTION
**Pull request by jhirvioja**
- Added PowerToys PowerRename random string values documentation text
  - **PR not yet merged**: https://github.com/microsoft/PowerToys/pull/32836

### CODE REVIEW

**Review by alvinashcraft — APPROVED** [review]
This looks good. I'll hold off on merging until the feature is released. Thanks!

File: hub/powertoys/powerrename.md [hunk 1]
```diff
@@ -110,6 +110,21 @@ A _Replace with_ text `Image_${padding=4;increment=2;start=10}_` would produce t
...
+### Random string values
+
+If selected, you can use the following patterns as part of the _Replace with_ text:
+
+| Variable pattern    | Explanation                                                                                |
+| :------------------ | :----------------------------------------------------------------------------------------- |
+| `${rstringalnum=X}` | Random string with uppercase letters, lowercase letters and 0-9 digits, customized length. |
+| `${rstringalpha=X}` | Random string with uppercase letters and lowercase letters, customized length.             |
+| `${rstringdigit=X}` | Random string with 0-9 digits, customized length.                                          |
+| `${ruuidv4=X}`      | Random UUID according to v4 specification.                                                 |
```
Comment (htcfreek) [inline hub/powertoys/powerrename.md]: Why `=X`? Isn't it fixed length by definition?

Comment (jhirvioja) [reply hub/powertoys/powerrename.md]: You are correct, good catch. Updated

File: hub/powertoys/powerrename.md [hunk 2]
```diff
@@ -110,6 +110,21 @@ A _Replace with_ text `Image_${padding=4;increment=2;start=10}_` would produce t
...
+If selected, you can use the following patterns as part of the _Replace with_ text:
+
+| Variable pattern    | Explanation                                                                                |
+| :------------------ | :----------------------------------------------------------------------------------------- |
+| `${rstringalnum=X}` | Random string with uppercase letters, lowercase letters and 0-9 digits, customized length. |
+| `${rstringalpha=X}` | Random string with uppercase letters and lowercase letters, customized length.             |
+| `${rstringdigit=X}` | Random string with 0-9 digits, customized length.                                          |
+| `${ruuidv4}`      | Random UUID according to v4 specification.                                                 |
+
+By default, random string values created are mixed case. You can adjust the generating behavior with the general [text formatting options that PowerRename provides](#text-formatting).
```
Comment (Jay-o-Way) [inline hub/powertoys/powerrename.md:124]: Hm, the word "[case](https://en.wiktionary.org/wiki/case)", by itself, can have a number of very different meanings. Is there a word that would be more specific?

Comment (alvinashcraft) [reply hub/powertoys/powerrename.md:124]: But I think "mixed case" is pretty unambiguous when used in a sentence about string values. Would making that phrase bold or italic help?

Comment (jhirvioja) [reply hub/powertoys/powerrename.md:124]: Saying `mixed letter case` could tie it more closely to the context

Comment (alvinashcraft) [reply hub/powertoys/powerrename.md:124]: @Jay-o-Way / @jhirvioja - This is released on 0.82.0. I'm going to merge this now with the current wording. Feel free to circle back with another PR if you think of a better way to phrase the info about mixed case strings.

**Pull request merged**